example/src/CheckoutView.js

```diff
@@ -86,42 +86,50 @@ const CheckoutView = () => {
     }
   };
 
+  const { config, paymentMethods, onConfigChanged } = usePaymentMethods();
```

Comment:
It is good practice to group all `useXXX` hooks in the beginning of the function Component - I'd suggest to put this one right after the line 36 :)  

This way it is clear from the beginning of the code from where this Component is using resources.

Reply:
👍

Reply:
Fixed in [17aae14](https://github.com/Adyen/adyen-react-native/pull/56/commits/17aae14ac8766f76a157d7198033915d4d4426e6)